packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -830,6 +833,17 @@ export class DashboardModel {
                                     url: url || '',
                                 },
                             };
+                        case DashboardTileTypes.SQL_CHART:
+                            return <DashboardSqlChartTile>{
+                                ...base,
+                                type: DashboardTileTypes.SQL_CHART,
+                                properties: {
+                                    ...commonProperties,
+                                    savedSqlUuid: '',
```

Comment:
depends on: https://github.com/lightdash/lightdash/issues/10693

Reply:
We now have unique slugs too -- are we going to keep doing these by uuid or eventually switch to slug?

Reply:
I believe by uuid for the tiles and perhaps by slug when sharing 👍